src/Symfony/Bundle/FrameworkBundle/Resources/config/console.php

```diff
@@ -0,0 +1,290 @@
+            ->tag('monolog.logger', ['channel' => 'messenger'])
+
+        ->set('console.command.messenger_setup_transports', SetupTransportsCommand::class)
+            ->args([
+                service('messenger.receiver_locator'),
+                abstract_arg('Receiver names'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:setup-transports'])
+
+        ->set('console.command.messenger_debug', DebugCommand::class)
+            ->args([
+                abstract_arg('Message to handlers mapping'),
+            ])
+            ->tag('console.command', ['command' => 'debug:messenger'])
+
+        ->set('console.command.messenger_stop_workers', StopWorkersCommand::class)
+            ->args([
+                service('cache.messenger.restart_workers_signal'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:stop-workers'])
+
+        ->set('console.command.messenger_failed_messages_retry', FailedMessagesRetryCommand::class)
+            ->args([
+                abstract_arg('Receiver name'),
+                abstract_arg('Receiver'),
+                service('messenger.routable_message_bus'),
+                service('event_dispatcher'),
+                service('logger'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:failed:retry'])
+
+        ->set('console.command.messenger_failed_messages_show', FailedMessagesShowCommand::class)
+            ->args([
+                abstract_arg('Receiver name'),
+                abstract_arg('Receiver'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:failed:show'])
+
+        ->set('console.command.messenger_failed_messages_remove', FailedMessagesRemoveCommand::class)
+            ->args([
+                abstract_arg('Receiver name'),
+                abstract_arg('Receiver'),
+            ])
+            ->tag('console.command', ['command' => 'messenger:failed:remove'])
+
+        ->set('console.command.router_debug', RouterDebugCommand::class)
+            ->args([
+                service('router'),
+                service('debug.file_link_formatter')->nullOnInvalid(),
+            ])
+            ->tag('console.command', ['command' => 'debug:router'])
+
+        ->set('console.command.router_match', RouterMatchCommand::class)
+            ->args([
+                service('router'),
+                tagged_iterator('routing.expression_language_provider'),
+            ])
+            ->tag('console.command', ['command' => 'router:match'])
+
+        ->set('console.command.translation_debug', TranslationDebugCommand::class)
+            ->args([
+                service('translator'),
+                service('translation.reader'),
+                service('translation.extractor'),
+                param('translator.default_path'),
+                abstract_arg('twig.default_path'),
+                abstract_arg('Translator paths'),
```

Comment:
> `<argument type="collection" />`

Well, it seems we can't use `abstract_arg()` here, or we should add a new `collection_abstract_arg()` or something else to set `[]` as default.

The tests can be solved setting `[]` for both collection arguments.

Reply:
@AhmedRaafat14 I afraid we need to change it again to `[], // Translator paths`, same for Twig paths argument (same for both commands), thanks!

Reply:
I was I did that it fixes the tests then I thought it might be me mistaken.

Thank you for confirming that I will update the PR ASAP.

Reply:
@javiereguiluz @yceruto 
To fix the testing I had to revert back every `collection` argument to an empty array `[]` with a comment.

Also even for the `console.command.translation_debug` the unit tests failed with using `abstract_arg('twig.default_path')` it throws an exception that it expects string but got an `AbstractArgument`.
I will give it sometime tomorrow to try to fix it :) if I can.

What do you think?

*Exception Example:*
```
Symfony\Component\DependencyInjection\Exception\InvalidParameterTypeException: 
Invalid definition for service "console.command.translation_debug": argument 5 of 
"Symfony\Bundle\FrameworkBundle\Command\TranslationDebugCommand::__construct" 
accepts "string", "Symfony\Component\DependencyInjection\Argument\AbstractArgument" passed.
```